bench/workgen/workgen.cxx

```diff
@@ -1567,17 +1614,29 @@ Operation::kv_compute_max(bool iskey, bool has_random)
     TableOperationInternal *internal = static_cast<TableOperationInternal *>(_internal);
 
     int size = iskey ? _key._size : _value._size;
-    if (size == 0)
-        size = iskey ? _table.options.key_size : _table.options.value_size;
+    if (size == 0) {
+        if (_random_table) {
+            const std::string err_msg("Cannot have a size of " + std::to_string(size) +
```

Comment:
You can say "Cannot have a size of 0 ..." since it is in the true part of the conditional. It is clearer to read that way.